### ISSUE TITLE
Failure when running npm run dist

### ISSUE THREAD

**Issue by westonruter** [Automattic/customize-direct-manipulation#21]
As instructed I tried running `npm install` and `npm run dist`. However, the latter ended with:

```
> customize-direct-manipulation@1.0.0 dist ...wordpress-develop/src/wp-content/plugins/customize-direct-manipulation
> grunt dist

(node) v8::ObjectTemplate::Set() with non-primitive values is deprecated
(node) and will stop working in the next major release.

...

Running "browserify:devPreview" (browserify) task
Fatal error: Invalid mapping: {"generated":{"line":468,"column":24},"source":"api.js","original":{},"name":null}

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node/6.4.0/bin/node" "/usr/local/bin/npm" "run" "dist"
npm ERR! node v6.4.0
npm ERR! npm  v3.7.1
npm ERR! code ELIFECYCLE
npm ERR! customize-direct-manipulation@1.0.0 dist: `grunt dist`
npm ERR! Exit status 3
npm ERR!
npm ERR! Failed at the customize-direct-manipulation@1.0.0 dist script 'grunt dist'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the customize-direct-manipulation package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     grunt dist
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs customize-direct-manipulation
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls customize-direct-manipulation
npm ERR! There is likely additional logging output above.
```

The `npm-debug.log` is as follows:

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/Cellar/node/6.4.0/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'run',
1 verbose cli   'dist' ]
2 info using npm@3.7.1
3 info using node@v6.4.0
4 verbose run-script [ 'predist', 'dist', 'postdist' ]
5 info lifecycle customize-direct-manipulation@1.0.0~predist: customize-direct-manipulation@1.0.0
6 silly lifecycle customize-direct-manipulation@1.0.0~predist: no script for predist, continuing
7 info lifecycle customize-direct-manipulation@1.0.0~dist: customize-direct-manipulation@1.0.0
8 verbose lifecycle customize-direct-manipulation@1.0.0~dist: unsafe-perm in lifecycle true
9 verbose lifecycle customize-direct-manipulation@1.0.0~dist: PATH: /usr/local/lib/node_modules/npm/bin/node-gyp-bin:...wordpress-develop/src/wp-content/plugins/customize-direct-manipulation/node_modules/.bin:/usr/local/Cellar/node/6.4.0/bin:...wordpress-develop/bin:/Users/$USER/bin:...phpcs/scripts:/usr/local/Cellar/php55/5.5.37/bin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
10 verbose lifecycle customize-direct-manipulation@1.0.0~dist: CWD: ...wordpress-develop/src/wp-content/plugins/customize-direct-manipulation
11 silly lifecycle customize-direct-manipulation@1.0.0~dist: Args: [ '-c', 'grunt dist' ]
12 silly lifecycle customize-direct-manipulation@1.0.0~dist: Returned: code: 3  signal: null
13 info lifecycle customize-direct-manipulation@1.0.0~dist: Failed to exec dist script
14 verbose stack Error: customize-direct-manipulation@1.0.0 dist: `grunt dist`
14 verbose stack Exit status 3
14 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/lifecycle.js:239:16)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at EventEmitter.emit (events.js:191:7)
14 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:24:14)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at ChildProcess.emit (events.js:191:7)
14 verbose stack     at maybeClose (internal/child_process.js:852:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
15 verbose pkgid customize-direct-manipulation@1.0.0
16 verbose cwd ...wordpress-develop/src/wp-content/plugins/customize-direct-manipulation
17 error Darwin 15.6.0
18 error argv "/usr/local/Cellar/node/6.4.0/bin/node" "/usr/local/bin/npm" "run" "dist"
19 error node v6.4.0
20 error npm  v3.7.1
21 error code ELIFECYCLE
22 error customize-direct-manipulation@1.0.0 dist: `grunt dist`
22 error Exit status 3
23 error Failed at the customize-direct-manipulation@1.0.0 dist script 'grunt dist'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the customize-direct-manipulation package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     grunt dist
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs customize-direct-manipulation
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls customize-direct-manipulation
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by sirbrillig** [comment]
Ugh. Thanks. Should be fixed by 43dd2756a3352f1761ad01847cd26e857218dadc

**Comment by westonruter** [comment]
Success. 👍 Fixed. 🙇
